services/web-server/src/graphql/WorkerManager.graphql

```diff
@@ -0,0 +1,25 @@
+type WMWorkerTypeSummary {
```

Comment:
Where can I find the worker manager schema?

Reply:
coming in future PRs

Reply:
> actually, which exactly schema are we talking about?

The file (schema) I'm referring to is where all of the properties of `WorkerManagerWorkerTypeSummary` are defined.

Reply:
https://github.com/taskcluster/taskcluster/pull/566/files#diff-4a88f9e6d2a378c002f9ae9f78eade9b this one?

Reply:
ok, so I think this is what you were asking about https://github.com/taskcluster/taskcluster/pull/566/files#diff-4a88f9e6d2a378c002f9ae9f78eade9b (it's part of this PR)

Reply:
in that schema, `description` is not required